ArduPlane/quadplane.cpp

```diff
@@ -1292,3 +1300,37 @@ bool QuadPlane::verify_vtol_land(const AP_Mission::Mission_Command &cmd)
     check_land_complete();
     return false;
 }
+
+void QuadPlane::enable_geofence(void)
```

Comment:
@magicrub good point. I will change enable_geofence() and disable_geofence() into a single function with a bool flag, something like
`void check_geofence_autoenable(bool enable_or_disable)`